cmds/exp/watch/watch.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2015-2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"fmt"
+	"flag"
+	"os"
+	"os/exec"
+	"time"
+	"bufio"
+
+)
+
+var (
+	t       	= flag.Bool("t", false, "Don't print header")
+	n       	= flag.Int64("n", 2, "Loop period in SEC, default 2")
+)
+
+func init() {
+	flag.Usage = func() {
+		fmt.Fprintf(os.Stderr, "watch [-n SEC] [-t] PROG ARGS\n")
+		fmt.Fprintf(os.Stderr, "Run PROG Periodically\n")
+
+		flag.PrintDefaults()
+	}
+}
+
+func main() {
+	flag.Parse()
+
+	argRem := flag.Args()
+
+	if len(argRem) == 0 {
```

Comment:
`if len(argRem) < 2`

Reply:
argRem is the remaining argument list after parsing the flags. (The ones that are not part of flags). After the flags are parsed, the remaining there should be at least 1 arg that will be the executable name to run